lib/utils/border_text_style.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:bordered_text/bordered_text.dart';
+import 'package:flutter/material.dart';
+
+class BorderTextStyle extends StatelessWidget {
+  final Text child;
+
+  const BorderTextStyle({
+    Key? key,
+    required this.child,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BorderedText(
```

Comment:
Really good way to reuse widgets. 👍🏼 
Think about make another folder for reusable widgets, utils are more functions than UI.